include/ua_plugin_securitypolicy.h

```diff
@@ -111,11 +111,12 @@ typedef struct {
      * key.
      *
      * @param securityPolicy the securityPolicy the function is invoked on.
-     * @param channelContext the channelContext which contains information about
-     *                       the keys needed to decrypt the message.
+     * @param securityPolicyContext the securityPolicyContext which contains
+     *                              information about the keys needed to decrypt
+     *                              the message.
      * @param data the data to decrypt. The decryption is done in place. */
     UA_StatusCode (*decrypt)(const UA_SecurityPolicy *securityPolicy,
-                             void *channelContext,
```

Comment:
This is a (good) simplification and a big bug-magnet (bad).
It is impossible to know for "the new guy" when to use the channelcontext and when to use the policycontext.

Currently, symmetric and asymmetric encryption are both a "cryptoModule" and share this API.
The symmetric encryption needs to channel context in this place.
We should differentiate their APIs to make the difference between channelcontext and policycontext visible.

**Can we move this change to a second PR?**

Reply:
Sure. I can add an issue for this change.